src/components/__tests__/Tabs-node-test.js

```diff
@@ -1,6 +1,3 @@
-/**
- * @jest-environment node
```

Comment:
Allowed this tests to run on `jsdom` so that `window` can be used. All tests keep working.

Reply:
Did accessing window crash the test? Or how did you notice this?

Reply:
Yes, it was because of having the tests failing on CI: https://travis-ci.org/github/reactjs/react-tabs/builds/753987091